### ISSUE TITLE
Test passes with different images when using 'seeVisualDiffForElement' for comparison.

### ISSUE THREAD

**Issue by Djet78** [codeceptjs/codeceptjs-resemblehelper#80]
Hello! 

I've got a test looking like that:
```
1 Scenario.only('Cross can be added', async (I, editorPage) => {
2    I.click(editorPage.tool.cross);
3    await I.addTool(50, 50);
4    I.saveElementScreenshot(editorPage.lastAddedTool, 'CROSS_ELEM.png');
5    I.seeVisualDiffForElement(editorPage.lastAddedTool, 'CROSS_ELEM.png',  {prepareBaseImage: false, tolerance: 0});
6 }).tag('editor').tag('tools').tag('cross');
```

If I change added tool (2nd line) test keep passing instead of fail. 
Both images have the same dimensions. 
Also, for some reason, test produces diff image, without highlighted difference.

## Test run results

Test Output image: ![OUTPUT_CROSS_ELEM](https://user-images.githubusercontent.com/38956861/94015406-7ad66400-fdb5-11ea-831c-8c4f829d0b35.png)

Base image: ![BASE_CROSS_ELEM](https://user-images.githubusercontent.com/38956861/94015662-d7398380-fdb5-11ea-82bd-1ffbd78543fe.png)

Generated diff image (generates copy of base image): ![Diff_CROSS_ELEM](https://user-images.githubusercontent.com/38956861/94015711-e6203600-fdb5-11ea-9591-7029e372ad80.png)

Console output:
![Screenshot](https://user-images.githubusercontent.com/38956861/94016464-d35a3100-fdb6-11ea-9e6d-f04001dda0f9.png)

## Additional info
* OS: Windows 10
* CodeceptJS version: 2.6.10
* codeceptjs-resemblehelper version: 1.9.2

Checked with different 'tolerance' level but got almost the same results. Diff image generated only with {tolerance: 0}
Also checked larger images - same results.

**Comment by isudilovskiy** [comment]
Same issue 
codeceptjs: 3.0.2
codeceptjs-resemblehelper: 1.9.3
playwright: 1.7.1

**Comment by shr2** [comment]
The main issue here is that the obvious differences between base and current image are not detected and the test is not failing.

**Comment by shr2** [comment]
Yes I did. Heres my package json
![image](https://user-images.githubusercontent.com/17846323/169820690-8bddd7ff-d8ff-4384-9485-6d4a131192e1.png)

**Comment by sharath2106** [comment]
We have released the latest version ([1.9.7](https://www.npmjs.com/package/codeceptjs-resemblehelper)) of the library yesterday. I would ask to check against the latest version and let us know.
